flask_api.py

```diff
@@ -90,11 +93,26 @@ def resample_audio():
     """Resample the audio file to adhere to the Nimbus audio sampling standard."""
     pass
 
-
-def save_audiofile():
-    """Actually save the file into Google Drive or whereever we may do it."""
-    pass
-
+def save_audiofile(filename, content):
+    """Actually save the file into Google Drive."""
+    # Initialize our google drive authentication object using saved credentials,
+    # or through the command line
+    gauth = GoogleAuth()
+    gauth.CommandLineAuth()
```

Comment:
Does this require command line input? Could we pipe in a credentials.txt file to automate the process?

Reply:
It does if you don't have a credentials.json and a settings.yaml file, which contain the client secret as well as an account token so I didn't upload them here.

Reply:
Okay sounds good! Later we can work those into Github Actions under a ``secret``

Reply:
Basically every file will be uploaded/owned by the csaicalpoly gmail account or one of our choosing.

Reply:
That's ideal, all of the user info will be included in the filename.